genai-perf/README.md

```diff
@@ -581,6 +581,35 @@ The standard deviation of height of images in pixels when generating synthetic i
 The compression format of the images. If format is not selected,
 format of generated image is selected at random.
 
+##### `--audio-length-mean <int>`
+
+The mean length of audio data in seconds. (default: `10`)
+
+##### `--audio-length-stddev <int>`
+
+The standard deviation of the length of audio data in seconds.
+(default: `0`)
+
+##### `--audio-format <str>`
```

Comment:
nit: as much as possible, try to keep this file alphabetized.
Aliases throw that off for sure and logical groupings do too. Taking that into consideration, I would reorder this small group.

Reply:
Done!